.github/workflows/build-images.yaml

```diff
@@ -54,14 +54,13 @@ on:
         value: ${{ jobs.tags.outputs.daemon }}
 
 env:
-  IMG_CONTROLLER:         "neondatabase/neonvm-controller"
-  IMG_VXLAN_CONTROLLER:   "neondatabase/neonvm-vxlan-controller"
-  IMG_RUNNER:             "neondatabase/neonvm-runner"
-  IMG_KERNEL:             "neondatabase/vm-kernel"
-  IMG_SCHEDULER:          "neondatabase/autoscale-scheduler"
-  IMG_AUTOSCALER_AGENT:   "neondatabase/autoscaler-agent"
-  IMG_DAEMON:             "neondatabase/neonvm-daemon"
-  IMG_CLUSTER_AUTOSCALER: "neondatabase/cluster-autoscaler-neonvm"
+  IMG_CONTROLLER:         "ghcr.io/neondatabase/neonvm-controller"
```

Comment:
we can left that unchanged and add string "ghcr.io" inside the step we need it.
That will lead to:
1. less changes and more readable PR
2. no need for the `"${image#ghcr.io/}` in the `Copy images to DockerHub step`

Reply:
Env vars for images don't include anything registry-dependent (parts for Docker Hub and GHCR are similar, but for ACR/ECR are pretty different).